x-pack/test/functional/apps/search_inference_management/inference_management.ess.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ModelConfig } from '@kbn/inference_integration_flyout/types';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common', 'inferenceManagement']);
+  const browser = getService('browser');
+  const ml = getService('ml');
+
+  /* Due to a race condition between registrating an app and license subscription, 
+    the FTR tests can not access the desired application.
+  */
+  describe.skip('ESS Inference Management UI', function () {
+    this.tags('skipMKI');
+
+    before(async () => {
+      await pageObjects.common.navigateToApp('home');
+      await pageObjects.common.navigateToApp('enterprise_search/overview');
+      await pageObjects.common.navigateToApp('inferenceManagement');
+    });
+
+    describe.skip('endpoint empty view', () => {
+      it('is loaded successfully', async () => {
+        await pageObjects.inferenceManagement.InferenceEmptyPage.expectComponentsToBeExist();
+      });
+      it('opens add inference flyout', async () => {
+        await pageObjects.inferenceManagement.InferenceEmptyPage.expectFlyoutTobeOpened();
+      });
+    });
+
+    describe.skip('endpoint tabular view', () => {
+      before(async () => {
+        const modelConfig: ModelConfig = {
+          service: 'elser',
+          service_settings: {
+            num_allocations: 1,
+            num_threads: 1,
+          },
+        };
+        await ml.api.createInferenceEndpoint('endpoint-1', 'sparse_embedding', modelConfig);
```

Comment:
This endpoint was created in the ‘before’ block. We may need to write some cleanup code in the ‘after’ block to delete the endpoint and remove the underlying trained model if it was added during the creation of the endpoint.

Reply:
At the last test, we removed the `inference endpoint` from the UI. Do we still need to do the cleanup or it will be handled through the application itself?

Reply:
Please check whether creation of inference endpoint adds any new trained model. If so, we need to clean up the trained model.